### PR TITLE
Fix create record test

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Anastasia Shemyakinskaya shem98a@gmail.com

### CODE REVIEW

File: src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java [hunk 1]
```diff
@@ -34,29 +35,29 @@ public void setup() {
...
-      new KafkaProducer<>(producer_properties, new StringSerializer(), new ByteArraySerializer());
+        new KafkaProducer<>(producer_properties, new StringSerializer(), new ByteArraySerializer());
     adminClient = KafkaAdminClient.create(properties);
   }
 
   @Test
   public void testCreateRecord() throws Exception {
     adminClient.createTopics(Collections.singletonList(new NewTopic(TOPIC_NAME, 1, (short) 1)));
-    final byte[] one_mb_of_data = new byte[999970];
+    final byte[] one_mb_of_data = new byte[900000];
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java]: 1. What is the purpose of this change?
2. Camel case naming should be used instead of the snake case for any java code.
3. The variable name ("1MB") seem to not represent the actual variable value :)

**Review by akurilov — CHANGES_REQUESTED** [review]

File: src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java (hunk 1 above)
Comment (AnastasiaShemyakinskaya) [reply src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java]: Value 999970 is too large to accept by kafka server and test fails with exception RecordTooLarge.

File: src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java [hunk 2]
```diff
@@ -11,52 +12,46 @@
...
-
-  @BeforeClass
-  public static void createContainers() {
-    try {
-      KAFKA_NODE_CONTAINER = new KafkaNodeContainer();
-    } catch (final Exception e) {
-      throw new AssertionError(e);
-    }
-  }
+  private static Timestamp timestamp;
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java]: the sql specific timestamp is not needed, just add the numeric value to the topic name

**Review by akurilov — CHANGES_REQUESTED** [review]

**Review by akurilov — APPROVED** [review]
